tests/operator_test.go

```diff
@@ -999,8 +999,7 @@ spec:
 		// running a VM/VMI using that previous release
 		// Updating KubeVirt to the target tested code
 		// Ensuring VM/VMI is still operational after the update from previous release.
-		It("[test_id:3145]from previous release to target tested release", func() {
-
+		It("[QUARANTINE][test_id:3145]from previous release to target tested release", func() {
```

Comment:
we shouldn't quarantine this one. this is a critical test that guarantees our compatibility.  we're going to have to figure otu what's going on here. Is there an issue about this test's flakiness yet?

Reply:
it looks like this is related to the timing of the live migration for workload updates. i'm looking at it. 

```
tests/operator_test.go:1002
Timed out after 320.000s.
All VMIs should update via live migration
Expected
    <*errors.errorString | 0xc001dfca50>: {
        s: "waiting for migration b391ddf8-a330-453a-a3a8-e4322ae8b3b4 to complete for vmi kubevirt-test-default1/testvmi-csx2n",
    }
to be nil
tests/operator_test.go:653
```

Reply:
> Is there an issue about this test's flakiness yet?

Indeed https://github.com/kubevirt/kubevirt/issues/3610

Reply:
i'm working on this here, https://github.com/kubevirt/kubevirt/pull/5159

Reply:
If David's changes have been cherry picked, can't we remove the QUARANTINE here?

Reply:
Good catch @dhiller  removed quarantine label.